receiver/influxdbreceiver/receiver.go

```diff
@@ -184,7 +184,7 @@ func (r *metricsReceiver) handleWrite(w http.ResponseWriter, req *http.Request)
 		err = batch.AddPoint(string(measurement), tags, fields, ts, common.InfluxMetricValueTypeUntyped)
 		if err != nil {
 			w.WriteHeader(http.StatusBadRequest)
-			_, _ = fmt.Fprintf(w, "failed to append to the batch")
+			_, _ = fmt.Fprintf(w, "failed to append to the batch: %w", err)
```

Comment:
```suggestion
			_, _ = fmt.Fprintf(w, "failed to append to the batch: %v", err)
```
Sorry govet caught my suggestion as faulty:
```
fmt.Fprintf does not support error-wrapping directive %w
```

Reply:
Ah - all good. Thanks for fixing